src/components/Multi/Route.js

```diff
@@ -92,7 +92,7 @@ function Geometry() {
                 _coordinates = coordinates
             }
             else {
-                throw new Error('not a array nigger')
+                throw new Error('Not an array, noob!')
```

Comment:
Please make error Not an array, contact administrator.

Reply:
done